packages/ruleset/src/functions/schema-type.js

```diff
@@ -1,17 +1,26 @@
-const { validateSubschemas } = require('../utils');
+const { mergeAllOfSchemaProperties, validateSubschemas } = require('../utils');
 
 module.exports = function(schema, _opts, { path }) {
   return validateSubschemas(schema, path, schemaDescription);
 };
 
 function schemaDescription(schema, path) {
```

Comment:
I'm realizing that I had a copy/paste error when I added this rule and probably should have named this `schemaType` 😅 

Do you mind making that change in this PR? All of your changes to this file look good 👍